lib/metro_2/records/base_segment.rb

```diff
@@ -50,5 +50,25 @@ class BaseSegment < Record
     alphanumeric_field :postal_code, 9
     alphanumeric_field :address_indicator, 1
     alphanumeric_field :residence_code, 1
+
+    def joint_segment=(joint_segment)
```

Comment:
This should be renamed `appendages` and allow for multiple different segments per base segment. This will allow furnishers to include a J1 segment and an L1 segment, for example. The usage would have to change slightly since you would be shoveling into an array instead of simple assignment.